docs/cristin.ttl

```diff
@@ -163,13 +173,21 @@
 
 :dato_opprettet
     a               :cristinfield ;
-    rdfs:comment    "The date the resource was registered in Cristin." ;
+    rdfs:comment    """
+    When the domain is :Result, it is the date the resource was registered in Cristin or its predecessor.
+    When the domain is :VITENSKAPELIGARBEID_LOKAL, it is the date the resource was registered in NVI.
```

Comment:
Well, not exactly, unfortunately, because nothing is ever individually registered in NVI.

All data is compiled in the Cristin database, checked and crosschecked, and then a report is generated once a year which is sent to NVI.

The date of the individual record is when that was created in the Cristin database, whether or not it eventually was sent to NVI. It's really basically only a database timestamp.

How about:

When the domain is :VITENSKAPELIGARBEID_LOKAL, it is the date the resource's NVI processing record for the respective institution was created.